pkg/minikube/download/preload.go

```diff
@@ -190,6 +193,38 @@ func Preload(k8sVersion, containerRuntime string) error {
 	return nil
 }
 
+var listTarContents = func(k8sVersion, containerRuntime string) (*bytes.Buffer, error) {
```

Comment:
It is safe to assume if there is a preload for a specific kubernetes version, then there is kubectl and other binaries inside of it, this operation is expensive and also wouldn’t work on non-Linux.

How about a simple check if the preload doesn’t exist for that k8s version or if the user explicitly specified —preload=false then we download those binaries ... kubeadm,kubectl,...

Reply:
Done!